ci/docker-compose.yml

```diff
@@ -17,19 +17,23 @@ services:
       CONJUR_ACCOUNT: cucumber
       CONJUR_DATA_KEY:
       RAILS_ENV:
+      REQUIRE_SIMPLECOV: "true"
       CONJUR_LOG_LEVEL: debug
       CONJUR_AUTHENTICATORS: authn-ldap/test,authn-ldap/secure,authn-oidc/keycloak
       LDAP_URI: ldap://ldap-server:389
       LDAP_BASE: dc=conjur,dc=net
       LDAP_FILTER: '(uid=%s)'
       LDAP_BINDDN: cn=admin,dc=conjur,dc=net
       LDAP_BINDPW: ldapsecret
+      WEB_CONCURRENCY: 0
+      RAILS_MAX_THREADS: 10
```

Comment:
I think we may want to document somewhere that the ci-based runs now will be single-threaded so that someone doesn't try to benchmark the app based on this setup.

Reply:
Any suggestions where?

Reply:
Hrmm... not sure tbh - maybe [here](https://github.com/cyberark/conjur#set-up-a-development-environment)?

Reply:
Added a note